trip_app/trips/tests/test_views.py

```diff
@@ -1,19 +1,23 @@
 from datetime import timedelta
```

Comment:
Look at https://testtools.readthedocs.io/en/latest/ , this can be useful for tests, have a lot of useful types of asserts and a lot yummy thing in there.

Reply:
Thanks, I will take a look at this!